cmake/do_abi_check.cmake

```diff
@@ -75,7 +75,9 @@ FOREACH(file ${ABI_HEADERS})
   FILE(REMOVE ${tmpfile})
   EXECUTE_PROCESS(
     COMMAND diff -w ${file}.pp ${abi_check_out} RESULT_VARIABLE result)
-  IF(NOT ${result} EQUAL 0)
+  IF(${result} MATCHES "No such file or directory")
```

Comment:
It's quite fragile to use unquoted `${var}` in cmake commands, it'll be simply substituted and split, and a semicolon will break the script. Use either
```cmake
   IF(result MATCHES "No such file or directory")
```
or
```cmake
   IF("${result}" MATCHES "No such file or directory")
```

Reply:
_sorry, only noticed it just now_

Reply:
Fixed in https://github.com/MariaDB/server/compare/0db68e8fe046ad9716ee49d8547023b6a7f16d94..51f5d5fbcae188880be37fea8444d215b38eaadc

Reply:
sorry, but you missed the second instance of it, in `ELSEIF(NOT ${result} EQUAL 0)`

Reply:
True, fixed now both rows, also the one that was there (wrongly) to from before.